Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:16
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+# for production
+# RUN npm ci --only=production
+
+COPY . .
+
+EXPOSE 3000
```

Comment:
Can we please run this on 80 & 443 too?
The ssl keys should be a docker parameter

Reply:
Fixed.

Reply:
Fixed the first point. 
Let's discuss more on the ssl point.